messages/action-summary-viewer.md

```diff
@@ -0,0 +1,15 @@
+# common.summary-header
+
+Summary
+
+# common.logfile-location
+
+Additional log information written to:
+
+# config-action.no-outfiles
+
+No output file was specified.
```

Comment:
Analogous to `run-summary-viewer.md`'s message "No results files were specified."